src/main/CommandLine.cpp

```diff
@@ -477,10 +477,9 @@ CommandLine::ConfigOption::getConfig(bool logToFile) const
     auto configFile =
         mConfigFile.empty() ? std::string{"stellar-core.cfg"} : mConfigFile;
 
-    LOG_INFO(DEFAULT_LOG, "Config from {}", configFile);
-
     // yes you really have to do this 3 times
     Logging::setLogLevel(mLogLevel, nullptr);
+    LOG_INFO(DEFAULT_LOG, "Config from {}", configFile);
```

Comment:
I think it would be better to do #1427 - since I opened that other issue, we moved to spdlog, and I see that we can just swap out `stdout_sink_mt` with `stderr_sink_mt` what do you think?

Reply:
Plausible. I suspect it'll break someone, somewhere, who relies on how things currently are. And I _suspect_ horizon is merging both streams for log-consumption currently. But I agree it's "more of the right thing", ish.

(I think? I mean .. there is no stdlog stream. Logs are not always errors. But stderr is from the "strong, silent" era of software where you only wasted teletype output when something went wrong)

Reply:
The concern with a change like the one in this PR is that it is brittle: if anybody later adds some logging in the wrong spot (potentially deep in the call stack), it will break clients that expect to find json.

I agree, doing it the other way can potentially break some clients but should be stable over time:
for clients that don't merge streams already, that break will be pretty obvious *and* we'll fix another problem with those clients as they're dropping on the floor whatever can end up in stderr like critical crash information when we abort (we use `std::fprintf(stderr, ` - some other libraries may do the same).

Reply:
@graydon - it's a "rip off the band-aid" type of situation. Let's just do it

Reply:
@MonsieurNicolas ok, band-aid ripped off.